src/Bicep.Core.Samples/Files/InvalidOutputs_CRLF/main.diagnostics.bicep

```diff
@@ -221,10 +221,22 @@ output noInnerLoopsInOutputs2 object = {
   }]
 }
 
+//KeyVault Secret Reference
+resource kv 'Microsoft.KeyVault/vaults@2019-09-01' existing = {
+  name: 'testkeyvault'
+}
+
+output keyVaultOutput string = kv.getSecret('mySecret')

```

Comment:
>getSecret [](http://example.com/codeflow?start=34&length=9)

Can you add a case where the function call is in deeper in an expression and also something involved complex objects?

Reply:
Fixed.